### PR TITLE
Fix grammatical error.

### PR DESCRIPTION
**Pull request by budhirajamadhav**
Fix grammatical issue.

### CODE REVIEW

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Containers cannot assume an identity from the host because the Security Account
 
 #### Memory reservations {#resource-management-memory}
 
-Windows does not have an out-of-memory process killer as Linux does. Windows always
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:66]: The original wording is idiomatic English @budhirajamadhav 

In my view, no change is needed.

Comment (budhirajamadhav) [reply content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:66]: Ok, no issues. But, in my view- The clear, the better.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I second the point made by @sftim . The sentence does not lack clarity from my perspective and I don't think this requires a change.

**Comment by divya-mohan0209** [comment]
I recommend the closing of this PR since I don't believe it to be lacking clarity. @sftim thoughts?

**Comment by sftim** [comment]
I'm afraid so @budhirajamadhav.

We _are_ open to thorough reviews and rewrites for any existing page, if that results in an improvement. Fixing actual errors (spelling mistakes, typos, punctuation) is OK when the existing page is obviously wrong, but otherwise please try to avoid proposing [trivial edits](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits).

/close

**Pull request closed without merging**